modules/modules.nix

```diff
@@ -140,6 +140,7 @@ let
     (loadModule ./services/emacs.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/flameshot.nix { })
     (loadModule ./services/fluidsynth.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/gammastep.nix { })
```

Comment:
```suggestion
    (loadModule ./services/gammastep.nix { condition = hostPlatform.isLinux; })
```

Reply:
How is this linux only compared to redshift?

Reply:
It uses systemd which is only available on Linux. Redshift doesn't have this line because it was added before the conditional module loading feature came to be. Eventually it would be nice to also add the condition to the redshift module.

Reply:
done